VK/VK/UserGroups/VC/UserGroupsTableViewController.swift

```diff
@@ -77,7 +142,14 @@ final class UserGroupsTableViewController: UITableViewController {
         forRowAt indexPath: IndexPath
     ) {
         guard editingStyle == .delete else { return }
-        groups.remove(at: indexPath.row)
-        tableView.deleteRows(at: [indexPath], with: .fade)
+        let group = groups[indexPath.row]
+        do {
+            let realm = try Realm()
```

Comment:
```suggestion
            let realm = try Realm()
            try realm.write {
                 realm.delete(group)
             }
```
Навошта ты ствараў сэрвіс для захоўвання калі тут напроста звяртаешся